jlm/hls/backend/rhls2firrtl/RhlsToFirrtlConverter.cpp

```diff
@@ -4045,7 +4050,11 @@ RhlsToFirrtlConverter::toString(const circt::firrtl::CircuitOp circuit)
   // Export FIRRTL to string
   std::string outputString;
   ::llvm::raw_string_ostream output(outputString);
-  auto status = circt::firrtl::exportFIRFile(module, output);
+  auto status = circt::firrtl::exportFIRFile(
+      module,
+      output,
+      100,
```

Comment:
same here

Reply:
Adde variable

Reply:
Changed to size_t targetLineLength = 100;